### PR TITLE
[#180362804] Logit flatline alerts

### PR DESCRIPTION
**Pull request by whi-tw**
What
----

Add script to create the content of alert yaml files for logit

The one alert I've created is for a flatline alert - if fewer than 1200 log entries are received in 10 minutes, the alert will fire.

This alert is sent to pagerduty. Currently the pagerduty alert is only for GOV.UK PaaS Email only - this is changed from within PagerDuty, though.

Also, I've fixed the `backtick` custom cop - it didn't allow me to do a thing (which it should have) so I have fixed that.

This is blocked by https://github.com/alphagov/paas-credentials/pull/242 - as currently the secrets are only present on my local machine.

How to review
-------------

- Code Review

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1406,6 +1406,7 @@ GEM
       hashdiff
 
 PLATFORMS
+  arm64-darwin-21
```
Comment (risicle) [inline Gemfile.lock:1409]: Yeah sorry I think this got removed as part of a reversion.

File: config/logit/alerts.d/flatline.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (risicle) [inline config/logit/alerts.d/flatline.yaml:1]: Suggest naming this file `.yaml.erb`

Comment (whi-tw) [reply config/logit/alerts.d/flatline.yaml]: I did originally, but then I have to strip it in the ruby and eugh. done now!

**Review by risicle — APPROVED** [review]
This appears to WFM.

Guess we need to apply this to all envs now then.

**Pull request merged**